### ISSUE TITLE
`0 of 0 changed or added relevant lines in 0 files are covered.` is awkward

### ISSUE THREAD

**Issue by jsoref** [lemurheavy/coveralls-public#1775]
https://github.com/thephantomthief/babelfish_extensions/pull/19#issuecomment-1878193976

> 0 of 0 changed or added relevant lines in 0 files are covered.

I'm having a hard time reading this message. I understand it's because all of the fields are 0s, but that doesn't mean that the message should be hard to read.

The message effectively suffers from division by zero.

The following message would yield as much information to a reader but cause less confusion:

> 0 changed or added relevant lines in 0 files are covered.

Even that is a bit much. The `changed or added relevant lines` part is still hard to parse when paired with even the single `0`.

It might be possible to simply write:

> 0 relevant lines in 0 files are covered.

or

> 0 changed lines in 0 files are covered.

and, beyond that... I'm not sure if `0 files` adds value. If there are 0 changed lines, they could come from an infinite number of files and it would still mean the same thing. Thus, I wonder if the message could be simplified to:

> 0 changed lines are covered.

---

Note: this is separate from #1743 which asks to fix the case of bad data when processing a PR that crosses forks.

**Comment by afinetooth** [comment]
@jsoref Thanks for the report. I have submitted a request to improve the wording in that scenario. I know we have an open ticket to improve copy in PR Comments in general, so it will probably be subsumed by that. No ETA yet, but I'll swing back if I get an update.
